src/app/processing/processing.component.ts

```diff
@@ -1,82 +1,34 @@
-import { Component, OnInit, Output, OnDestroy, DoCheck } from '@angular/core';
-import { ActivatedRoute, Params } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
 
-import { KnowledgeBaseService } from 'app/entry/knowledge-base/knowledge-base.service';
-import { ProcessingService } from './processing.service';
-import { ModalsService } from '../modals/modals.service';
-import { ProcessingArchitectureService } from '../services/processing-architecture.service';
-import { SidStatusService } from '../services/sid-status.service';
 import { ProcessingModel } from '@api/models';
-import { ProcessingApi } from '@api/services';
 
 @Component({
   selector: 'app-processing',
   templateUrl: './processing.component.html',
   styleUrls: ['./processing.component.scss']
 })
 export class ProcessingComponent implements OnInit {
+  processing: ProcessingModel;
   sections: any;
-  section: { id: number, title: string, short_help: string, items: any };
-  item: { id: number, title: string, evaluation_mode: string, short_help: string, questions: any };
+  currentSection: number;
```

Comment:
this field could not be of type `number` because you assign it an item of `sections` in method `getSectionById`. (please have a look at the interface `Section` in this PR for example `github.com/pia-lab/pialab/pull/47/commits/7ad0398e58eefbd50f40b3376895ccb4d353e480#diff-199c3a3dc71f64073bb3d9feaa2ac66c`